src/styles/blocks/nav.scss

```
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    margin: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+    display: inline-block;
+  }
+
+  &__item:last-child {
```

```suggestion
  &__item {
    &:last-child {
      any
 }
```
так по чек листу нужно писать абсолютно везде , не критично но все же